src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.UnitTests/Controllers/AuthorizationControllerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Controllers;
+using AutoFixture;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Application.Queries.CanAccessCohort;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    public class AuthorizationControllerTests
```

Comment:
Do you think it's worth having a test to assert that the correct value is returned from the action method?